index.html

```diff
@@ -1177,6 +1192,28 @@ <h2>Mapping for devices that do not support hover</h2>
             </div>
         </section>
     </section>
+    <section class='informative'>
+        <h2>Converting between <code>tiltX</code> / <code>tiltY</code> and <code>altitudeAngle</code> / <code>azimuthAngle</code></h2>
+        <p>Depending on the specific hardware and platform, user agents will likely only receive one set of values for the transducer orientation relative to the screen plane - either <code>tiltX</code> / <code>tiltY</code> or <code>altitudeAngle</code> / <code>azimuthAngle</code>. The following basic code provides an initial suggested approach for converting these values.</p>
+        <pre id="example_12" class="example" title="Converting between tiltX/tiltY and altitudeAngle/azimuthAngle"><code>/* Convering azimuthAngle / altitudeAngle to tiltX / tiltY */
+
+tiltX = Math.atan(Math.sin(azimuthAngle) / Math.tan(altitudeAngle)) * 180 / Math.PI;
+tiltY = Math.atan(Math.cos(azimuthAngle) / Math.tan(altitudeAngle)) * 180 / Math.PI;;
+
+/* Convering tiltX / tiltY to azimuthAngle / altitudeAngle */
+
+azimuthAngle = 0;
+tiltX = tiltX * Math.PI / 180;
+tiltY = tiltY * Math.PI / 180;
+if (tiltX != 0) {
+  azimuthAngle = Math.PI/2 - Math.atan2(-Math.cos(tiltX) * Math.sin(tiltY), Math.cos(tiltY) * Math.sin(tiltX));
+  if (azimuthAngle &lt; 0) {
+    azimuthAngle += 2*Math.PI;
+  }
+}
+altitudeAngle = Math.PI/2 - Math.acos(Math.cos(tiltX) * Math.cos(tiltY));</code>
```

Comment:
I did some back of the envelope calculations and my equations are different from these.
I tried some sanity checks on these but they don't add up. First the formulas are not reversible. I mean if I use a given tiltX,tiltY and change it to azimuth,altitude and again change it back I won't get the same tiltX,tiltY. 

Also the boundary cases that are easy to calculate with mind doesn't match either. So let's first agree on those as those also show the origin of these domains and whatnot.

So in my mind these are equivalent. What do you think  @patrickhlauke, @smaug----  @grorg @graouts
tiltX=0, tiltY=0 ===== altitude= pi/2, azimuth=0 (but technically could be anything as this is pen standing straight)
tiltX=90, tiltY=0 ===== altitude = 0, azimuth=0 (this is pen laying on the X axis its side completely touching the surface)
tiltX=0, tiltY=90 ====== altitude=0, azimuth=pi/2 (this is pen laying on the Y axes and its side completely touching the surface)
tiltX=90, tiltY=90 ====== altitude=0, azimuth=3*pi/2


These values are based on the assumption that azimuth is going to be 0 on X axis and going to increase on counterclock-wise rotation when looking at the plane from the top. Also the X axis is in the right direction and y axis is towards the top of the screen and Z axis is coming out of the plane (like right-handed coordinate system shown on the graphics in the spec).

However, the current Chrome implementation is a left handed one for tiltX and tiltY. i.e. right is positive X and down is positive Y. It does seem inline with @patrickhlauke [demo](https://patrickhlauke.github.io/touch/pen-tracker/) graphics. Unfortunately FF showed zero for tiltX/Y on my Surface book I have for testing. Not sure how the old Edge used to behave. So I believe It makes sense to change the spec in that case to match Chrome as the only implementation on that (i.e. X axis to the right of the screen, Y axis to the bottom and Z coming out of the plane). If we do so the expectation above regarding the transformation between the two coordinate system also change. So let's first decide on this.